### DISCUSSION TITLE
Status 401 not authorized error

### DISCUSSION THREAD

**Discussion by fliberdygibits** [luigi311/JellyPlex-Watched#209] [General]
I had this up and running yesterday evening between two servers.  I've not changed anything but today I'm getting the following error.

https://pastebin.com/gpSTRZug

I'm running 10.9.10 in an LXC and 10.9.11 in a VM, both in proxmox.

I've double and triple checked the only two things that seem they would be related to connecting to the server:  The jellyfin baseurl and the API for each.   I've also generated a new API key just to be safe.

**Comment by luigi311** [comment]
Do you have anything set to limit access such as the lan network? I know a few people had their lan network defined and this was running into issues since the container being used to run this was on a different subnet.
